From 303ff72a3b94cc10820d4117e8f4ac07d6dc0022 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 25 Jun 2018 22:24:33 +0300
Subject: [PATCH 4102/5242] MLK-18683-1 arm64: dts: imx8mm-evk: Fix
 misspelling stdout-path

commit  94bd986daa658ae037afb100072f122751b3a07e from
https://source.codeaurora.org/external/imx/linux-imx.git

The property is called "stdout-path" not "stdout-patch"

Fixes: c5454672bc06 ("MLK-18205-3 ARM64: dts: freescale: add i.MX8MM dtb")

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index ed489e9..f74a5fd 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -22,7 +22,7 @@
 
 	chosen {
 		bootargs = "console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200";
-		stdout-patch = &uart2;
+		stdout-path = &uart2;
 	};
 
 	leds {
-- 
1.7.9.5

